bot/exts/fun/uwu.py

```diff
@@ -0,0 +1,154 @@
+import random
+import re
+from functools import partial
+from typing import Callable
+
+from discord.ext import commands
+from discord.ext.commands import Cog, Context, clean_content
+
+from bot.bot import Bot
+from bot.exts.fun.fun import Fun
+from bot.utils import helpers
+
+
+# Wepwacement
+WEPWACE_HASH = {
+    "small": "smol",
+    "cute": "kawaii~",
+    "fluff": "floof",
+    "love": "luv",
+    "stupid": "baka",
+    "idiot": "baka",
+    "what": "nani",
+    "meow": "nya~",
+    "roar": "rawrr~",
+}
+
+EMOJI_LUT = [
+    "rawr x3",
+    "OwO",
+    "UwU",
+    "o.O",
+    "-.-",
+    ">w<",
+    "(⑅˘꒳˘)",
+    "(ꈍᴗꈍ)",
+    "(˘ω˘)",
+    "(U ᵕ U❁)",
+    "σωσ",
+    "òωó",
+    "(///ˬ///✿)",
+    "(U ﹏ U)",
+    "( ͡o ω ͡o )",
+    "ʘwʘ",
+    ":3",
+    ":3",  # important enough to have twice
+    "XD",
+    "nyaa~~",
+    "mya",
+    ">_<",
+    "😳",
+    "🥺",
+    "😳😳😳",
+    "rawr",
+    "uwu",
+    "^^",
+    "^^;;",
+    "(ˆ ﻌ ˆ)♡",
+    "^•ﻌ•^",
+    "/(^•ω•^)",
+    "(✿oωo)",
+]
+
+wepwace_regex = re.compile(r"(?<![w])[lr](?![w])")
+
+
+def _word_replace(input_string: str) -> str:
+    for word in WEPWACE_HASH:
+        input_string = input_string.replace(word, WEPWACE_HASH[word])
+
+    return input_string
+
+
+def _char_replace(input_string: str) -> str:
+    return wepwace_regex.sub("w", input_string)
+
+
+# Stuttering
+stutter_regex = re.compile(r"(\s)([a-zA-Z])")
+stutter_subst = "\\g<1>\\g<2>-\\g<2>"
```

Comment:
I think it takes up a whole lot of vertical space for very little benefit when we put these regexes inbetween the helper functions instead of grouping them all together at the top. Yeah yeah, it has better locality, but it just makes this file so much more bloated.

I suggest we move all of these regexes and `subst` variables up below the constants, all together in a block. I don't think that will really make this harder to work with, but it'll make the file look better.